builder/proxmox/common/builder.go

```diff
@@ -176,10 +176,17 @@ func getVMIP(state multistep.StateBag) (string, error) {
 
 	for _, iface := range ifs {
 		for _, addr := range iface.IpAddresses {
-			if addr.IsLoopback() {
+			if addr.To4() == nil && addr.To16() == nil {
+				// no IPv4 or IPv6 at all
 				continue
-			}
-			if addr.To4() == nil {
+			} else if addr.IsLinkLocalUnicast() || addr.IsMulticast() || addr.IsLoopback() {
+				// SiteLocalUnicast: fec0::/10
+				// multicast: ff00::/8
+				// multicast interface ff01::1
```

Comment:
these two have no special meaning to the purpose of this code, and they are already covered by `multicast: ff00::/8`, so drop this line and the next?